### PR TITLE
Pod Wars Game Mode

### PR DESCRIPTION
**Pull request by Kyle2143**
## About the PR 
This adds a new gimmick admin game mode. No plans to ever enter this type of mode into standard rotation. Does a few other changes to ancillary things, and I take quite a few liberties in terms of how things work in the actual mode. 

Here's the gist of the mode:
- Two teams, **NanoTrasen** and **Syndicate**. Each have a medium sized base ship with only the essentials on either end of the map that they must defend from the enemy team.
- Each team starts with 100 points, and you can gain/lose points by destroying the enemy ship or killing enemy players.
- Each base ship has a mining magnet that you can set up to mine easily, and also all the asteroids on the map are just packed to the brim with materials. The base ships have special manufacturers that let you build pods and a couple other things helpful to combat.
- At ~15 minutes in, **3 Control Points** on the map open up that teams will fight over to get useful loot.
- Each team's base ship has 4 "Critical Ship Systems" that are on the outside of the ship that can be destroyed for massive points. (Eventually the plan is to have each of these systems have tangible affects on the game mode outside of point loss)
- Each team has a Commander and the rest are all "Pod Pilots" who spawn with a space suit and some oxygen as well as a solid blaster pistol and a melee weapon(machete).
- Anyone who dies will be automatically cloned by the cloners on their ship at hyper speed, but your team will lose points.

------------------------------------------------------------------
This is basically covered above, but to break it down conceptually, there are 2 gameplay phases in the mode:
1. Mining/Building/Support - The map is big, to get anywhere you're gonna need a pod. While you can auto-fabricate some weak pods, to make anything good at killing other pods or the enemy base, you'll need to get better materials. 
2. Point Control/Combat - Along the center of the map there are a couple Space Stations/Ships that you can capture after the 15 minute mark, right now they're rather bare bones with only one offering intrinsic utility. But upon capturing them, your team's pods can warp to the beacons there, and every ~5 minutes held by a team, they will deposit a crate of some useful items.

----------------------------------------------------
Adds a whole new set of spacesuits for NanoTrasen pilots and their commander made by Flaborized. 
Adds a bunch of voice lines to assist with keeping players on track in the mode. The role of the **Syndicate Commander** is reprised by _Efrem_, and introducing _Phuzzy_ in the role of the **NanoTrasen commander**. Not to mention a bunch of new sound effects made for things in the mode and sound mixing done by Virvatuli.

## Why's this needed? 

Would be nice to have a sort of pure combat, with even teams, sort of thing to play every now and then. 

---------------------------------Technical Notes-------------------------------
- This is basically all done, and ready to test on the live server. All that's left to do is make an in-game guide for players when they spawn in like antag popups and to adjust some manufacturer item recipie material requirements. There's a lot of more possibilities to add, but I'm calling this done for now. 
- As for the code quality itself, I did do quite a few lazy hacks such as using #ifdefs checking for the map type and such, I removed most of em, but yeah. 
- I see quite a lot of files changed in this PR, more than I expected. I think I might have an issue with some merge, so I'll investigate that tomorrow, partly why this has the DNM tag. Should also give me a chance to clean up any obvious mistakes before Zewaka, Mordent, and Pali beat me up for it.

P.S. I also put the do not merge tag on this for now, please don't merge/play it before I'm aware. I'd like to be around for the initial round.

### CODE REVIEW

**Comment by DimWhat** [comment]
The NT commander sounds suspiciously like the Discount Dan's advertisement...

**Comment by Gannets** [comment]
Happy to finally see this pr! Good job!

**Comment by Eagletanker** [comment]
Holy shit this is cool!

File: _std/__build.dm [hunk 1]
```diff
@@ -106,8 +106,9 @@ o+`        `-` ``..-:yooos-..----------..`
 //#define MAP_OVERRIDE_ICARUS
 //#define MAP_OVERRIDE_GEHENNA			// Warcrimes WIP do not use
 //#define MAP_OVERRIDE_PAMGOC			// Pamgoc
-//#define MAP_OVERRIDE_WRESTLEMAP		// Wrestlemap by Overtone
-
+//#define MAP_OVERRIDE_WRESTLEMAP   // Wrestlemap by Overtone
+#define MAP_OVERRIDE_POD_WARS   // 500x500 Pod Wars map
```
Comment (ZeWaka) [inline _std/__build.dm:110]: xx

File: _std/macros/antag_popups.dm [hunk 2]
```diff
@@ -79,6 +79,9 @@
 // mindwipe from cloner zap chance
 #define SHOW_MINDWIPE_TIPS(M) M.Browse(grabResource("html/mindwipe.html"), MINDWIPE_TIPS_WINDOW)
 
+// mindwipe from cloner zap chance
```
Comment (ZeWaka) [inline _std/macros/antag_popups.dm:82]: fix

File: code/WorkInProgress/AnimationLibrary.dm [hunk 3]
```diff
@@ -1070,7 +1070,7 @@ proc/muzzle_flash_any(var/atom/movable/A, var/firing_angle, var/muzzle_anim, var
 	animate(transform = MD, time = 1, loop = -1, easing = LINEAR_EASING)
 
 // these don't use animate but they're close enough, idk
-/proc/showswirl(var/atom/target)
+/proc/showswirl(var/atom/target, var/play_sound = 1)
```
Comment (ZeWaka) [inline code/WorkInProgress/AnimationLibrary.dm:1073]:
```suggestion
/proc/showswirl(var/atom/target, var/play_sound = TRUE)
```

File: code/area.dm [hunk 4]
```diff
@@ -5175,3 +5175,251 @@ area/station/security/visitation
 
 
 */
+/// pod_wars Areas
```
Comment (ZeWaka) [inline code/area.dm:5178]:
```suggestion
// pod_wars Areas
```

File: code/turf/turf.dm [hunk 5]
```diff
@@ -993,6 +993,17 @@
 			playsound(src, "sound/impact_sounds/Generic_Stab_1.ogg", 50, 1)
 			T.build(src)
 
+#ifdef MAP_OVERRIDE_POD_WARS
```
Comment (ZeWaka) [inline code/turf/turf.dm:996]: won't work with the else, you need this to be `#if defined(x)`

File: code/obj/machinery/clonepod.dm [hunk 6]
```diff
@@ -60,6 +60,7 @@
 	var/static/list/sounds_function = list('sound/machines/engine_grump1.ogg','sound/machines/engine_grump2.ogg','sound/machines/engine_grump3.ogg',
 	'sound/impact_sounds/Metal_Clang_1.ogg','sound/impact_sounds/Metal_Hit_Heavy_1.ogg')
 
+	var/perfect_clone = 0		//if TRUE, then clones always keep normal name and recieve no health debuffs
```
Comment (ZeWaka) [inline code/obj/machinery/clonepod.dm:63]:
```suggestion
	var/perfect_clone = FALSE		//if TRUE, then clones always keep normal name and receive no health debuffs
```

File: code/modules/medical/surgery_tools.dm [hunk 7]
```diff
@@ -925,6 +936,10 @@ CONTAINS:
 				target.visible_message("<span class='success'>[owner] [vrb]es [owner == target ? "[his_or_her(owner)]" : "[target]'s"] wounds closed with [tool].</span>",\
 				"<span class='success'>[owner == target ? "You [vrb]e" : "[owner] [vrb]es"] your wounds closed with [tool].</span>")
 				repair_bleeding_damage(target, 100, repair_amount)
+				if (brute_heal)
+					random_brute_damage(target, brute_heal)
+				if (burn_heal)
+					random_burn_damage(target, burn_heal)
```
Comment (ZeWaka) [inline code/modules/medical/surgery_tools.dm:942]: DAMAGE????

File: code/datums/gamemodes/pod_wars.dm [hunk 8]
```diff
@@ -0,0 +1,3106 @@
+#define TEAM_NANOTRASEN 1
```
Comment (ZeWaka) [inline code/datums/gamemodes/pod_wars.dm:1]: PLEASE consider splitting this up into a folder

**Review by ZeWaka — CHANGES_REQUESTED** [review]

File: _std/__build.dm (hunk 1 above)
Comment (Kyle2143) [reply _std/__build.dm:110]: Woops...

File: code/datums/gamemodes/pod_wars.dm (hunk 8 above)
Comment (Kyle2143) [reply code/datums/gamemodes/pod_wars.dm:1]: Yeah, I should shouldn't I? I don't know why I haven't, just being lazy I guess. It was starting to get _really_ messy/annoying at the end.

File: code/modules/medical/surgery_tools.dm (hunk 7 above)
Comment (Kyle2143) [reply code/modules/medical/surgery_tools.dm:942]: Oof.

**Pull request merged**